beard/clustering/blocking_funcs.py

```diff
@@ -493,3 +494,38 @@ def block_single(X):
         equals zero.
     """
     return np.zeros(len(X), dtype=np.int)
+
+
+def block_last_name_first_initial(X):
```

Comment:
Could we base this function on name functions defined in `utils/names.py`? 

See also discussion in #20